src/include/log.h

```diff
@@ -84,8 +84,31 @@ extern "C" {
 #define	sys_printf(...)	syslog(LOG_NOTICE, __VA_ARGS__);
 #ifdef	DEBUG
 #define	DBPRT(x)	printf x;
-#else
-#define	DBPRT(x)
+#endif
+#ifdef DBPRT_LOG
+#include <stdlib.h>
+#include <errno.h>
+#include "libutil.h"
+#define STRIP_PARENS(...) __VA_ARGS__
+#undef DBPRT
+#define	DBPRT(x) \
+	if (will_log_event(PBSEVENT_DEBUG3)) { \
```

Comment:
Maybe instead of using DEBUG3, we create a new DEBUG level called PBSEVENT_DEBUGPRT or something like that.  This way it won't be interspersed with the normal DEBUG3 messages.

Reply:
Done in 43931b90596f6c47652a84f1981a4de00c305ff7.  This change does allow the level value to be overridden by compiler flags or a temporary addition to source and/or header files in order to make developer's desire of "I want the output now!" while debugging as simple as possible and yet minimize the impact on others.